pkg/storaged/utils.js

```diff
@@ -460,7 +460,7 @@ export function get_available_spaces(client) {
 export function prepare_available_spaces(client, spcs) {
     function prepare(spc) {
         if (spc.type == 'block')
-            return cockpit.resolve(spc.block.path);
+            return Promise.resolve(spc.block.path);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)